### PR TITLE
Issue 5006 - UI - LDAP editor tree not being properly updated

### PR DESCRIPTION
**Pull request by mreynolds389**
Description:  

Deleting an entry was the tree view did not update the treeview.  Updates to table view were also not seen in the tree view.  The views should now be in synch

Also, replaced some console logging with our "log_cmd" function in the editor utils file.

relates: https://github.com/389ds/389-ds-base/issues/5006

### CODE REVIEW

File: src/cockpit/389-console/src/lib/ldap_editor/treeView.jsx [hunk 1]
```diff
@@ -113,6 +113,10 @@ class EditorTreeView extends React.Component {
 
         this.handleNodeOnClick = (treeViewItem) => {
             console.log('handleNodeOnClick - treeViewItem: ', treeViewItem);
+            if (treeViewItem.dn === this.state.entryDn) {
```
Comment (droideck) [inline src/cockpit/389-console/src/lib/ldap_editor/treeView.jsx]: Weirdly enough, a browser crashes here when I:

1. Go to the Tree view and select a user under ou=People;
2. Go to the Table view and select the same user there;
3. Go back to the Tree view and open the same user view (may be an unnecessary step)
4. Go back to the Table view and delete the user.

Result - Browser crashed.

**Review by droideck — CHANGES_REQUESTED** [review]

File: src/cockpit/389-console/src/lib/ldap_editor/treeView.jsx (hunk 1 above)
Comment (mreynolds389) [reply src/cockpit/389-console/src/lib/ldap_editor/treeView.jsx]: I can not reproduce the crash

Comment (mreynolds389) [reply src/cockpit/389-console/src/lib/ldap_editor/treeView.jsx]: fixed

**Review by droideck — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**